site/content/en/docs/Advanced/limiting-resources.md

```diff
@@ -19,14 +19,8 @@ Kubernetes documentation for more details on "requests" and "limits" to both CPU
 
 ## GameServers
 
-{{% feature expiryVersion="1.9.0" %}}
```

Comment:
looks like we missed this one on the last release. ;)

Reply:
I didn't even notice as I was going through!